internal/string_helpers.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+import "strings"
+
+func HasAnyOfPrefixes(input string, prefixes ...string) bool {
```

Comment:
Good call moving this here 👍

Reply:
nit: doc comment

Reply:
it was good where it was at, but then I made use of it in another package. I'll update the doc string